src/getTime.js

```diff
@@ -18,7 +18,14 @@
  * @returns {string} - valid time
  */
 function getTime(str) {
-  // write code here
+  const regexTime = /\b(0[0-9]|1[0-9]|2[0-3]):[0-5][0-9]\b/g;
+  const time = str.match(regexTime);
+
+  if (time === null) {
+    return '';
+  }
+
+  return time[0];
```

Comment:
Dmytro told me that we can use a ternary operator in this task.

Reply:
OK cool :)